src/components/modal/PasswordSuccessfulModal.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { expect, test } from 'vitest';
+import { render, fireEvent } from '@testing-library/react';
+import PasswordSuccessfulModal from './PasswordSuccessfulModal';
+
+test('renders the modal when show is true', () => {
+    const { getByText } = render(
+      <PasswordSuccessfulModal show={true} onClose={() => {}} />
+    );
+    expect(getByText('Password Successfully Updated!')).toBeInTheDocument();
+    expect(
+      getByText(
+        'Your password has been successfully updated! You can now log in with your new password.'
+      )
+    ).toBeInTheDocument();
+  });
+  
+  test('close the modal when continue is clicked', async () => {
+    let modalClosed = false;
+    const handleClose = () => {
+      modalClosed = true;
+    };
+    const { getByText } = render(
+      <PasswordSuccessfulModal show={true} onClose={handleClose} />
+    );
+  
+    await fireEvent.click(getByText('Continue'));
```

Comment:
Consider using the [userevent api](https://testing-library.com/docs/user-event/intro/) instead